README.md

```diff
@@ -24,6 +24,27 @@ them are as follows:
     openldap_serverlocation: Portland
     openldap_serverorganization: IT
 
+    # Create users
+    openldap_enable_user_creation: true
+    openldap_server_users:
+      staff:
+        william:
+          password: 26April1564
+          fullname: William Shakespeare
+          firstname: William
```

Comment:
I think it may make sense to change this to givenname  so it matches the ldap schema key